<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorOverride</class>
 <widget class="QDialog" name="ColorOverride">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Override</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="coVerticalLayout">
    <item>
     <widget class="QGroupBox" name="coGroupBox">
      <property name="title">
       <string>Choose color:</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>149</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="coVerticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="defaultColorHorizontalLayout">
          <item>
           <widget class="QPushButton" name="defaultColor0Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultColor1Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 0)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultColor2Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultColor3Button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(128, 128, 128)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="sliderHorizontalLayout">
          <item>
           <widget class="QPushButton" name="sliderButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderHorizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonHorizontalLayout">
          <item>
           <widget class="QPushButton" name="overrideButton">
            <property name="text">
             <string>Override</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
